services/worker-manager/schemas/v1/workertype-list

```diff
@@ -0,0 +1,27 @@
+$schema: "/schemas/common/metaschema.json#"
+title: WorkerType Full
+description: A complete workerType definition
+type: object
+properties:
+  data:
```

Comment:
Let's have this key be `workertypes` to keep the same format as the list of clients from auth has for instance.